west.yml

```diff
@@ -142,7 +142,7 @@ manifest:
         - hal
     - name: hal_telink
       url: https://github.com/telink-semi/hal_telink
-      revision: ee363b2e82f2c06be7152530c3287b14522ea3b9
+      revision: 4ec4a267591b10ec652551e57bd5550c7fd3de37
```

Comment:
Please do not provide such changes! This revision is useless.

Reply:
done